dbms/src/Common/MemoryTracker.cpp

```diff
@@ -69,6 +69,20 @@ void MemoryTracker::alloc(Int64 size, bool check_memory_limit)
       *  we allow exception about memory limit exceeded to be thrown only on next allocation.
       * So, we allow over-allocations.
       */
+    Int64 current_amount = amount.load(std::memory_order_relaxed);
+    if (unlikely(accuracy_diff && real_rss > accuracy_diff + current_amount))
```

Comment:
only amount of root mem tracker is meaningful

Reply:
Oh sorry, I forgot it.